bodhi/server/util.py

```diff
@@ -1162,15 +1164,15 @@ def call_api(api_url, service_name, error_key=None, method='GET', data=None):
         base_error_msg = (
             'Bodhi failed to send POST request to {0} at the following URL '
             '"{1}". The status code was "{2}".')
-        rv = requests.post(api_url,
-                           headers={'Content-Type': 'application/json'},
-                           data=json.dumps(data),
-                           timeout=60)
+        rv = http_session.post(api_url,
+                               headers={'Content-Type': 'application/json'},
+                               data=json.dumps(data),
```

Comment:
FYI there's a shortcut for this in requests, the ``json=`` kwarg[0], which will handle serialization, headers, etc. This should be fine as is, just so you know.

[0] http://docs.python-requests.org/en/master/user/quickstart/#more-complicated-post-requests

Reply:
Ah nice! I'm inclined to leave it as-is since this PR is more about using the pooling, but thanks for letting me know! Feel free to send a PR that adjusts it if you like, or I might do it later myself :)